aggregator/src/main/scala/com/stratio/sparkta/aggregator/DataCube.scala

```diff
@@ -65,50 +63,19 @@ case class DataCube(dimensions: Seq[Dimension],
         dimension <- dimensions
         value <- event.keyMap.get(dimension.name).toSeq
         (bucketType, bucketedValue) <- dimension.bucketer.bucket(value)
-      } yield DimensionValue(DimensionBucket(dimension, bucketType), convertToBucketTypeOp(bucketType,bucketedValue))
+      } yield DimensionValue(DimensionBucket(dimension, bucketType),
+          TypeOp.transformValueByTypeOp(bucketType.typeOp, bucketedValue))
```

Comment:
I would suggest to rename the name of this class to DataCubeHelper instead of TypeOp and move to helpers package. As well, I would create a spec to this helper. It is not necessary now but I think that it will be useful: Up to you!